pkg/grpc/auditlogstore/protos/audit_log_store.pb.go

```diff
@@ -22,7 +22,8 @@ import fmt "fmt"
 import math "math"
 
 import (
-	context "golang.org/x/net/context"
```

Comment:
oh this is why it compiles. unfortunately it's not a good idea to edit this file, it's generated automatically by running `protoc --go_out=plugins=grpc:. pkg/grpc/auditlogstore/protos/audit_log_store.proto` (after installing `protoc` on your laptop). You can follow the instructions at http://www.grpc.io/docs/quickstart/go.html if you're curious